src/huggingface_hub/hub_mixin.py

```diff
@@ -212,20 +212,39 @@ def __init_subclass__(
         # Will be reused when creating modelcard
         tags = tags or []
         tags.append("model_hub_mixin")
-        cls._hub_mixin_info = MixinInfo(
-            model_card_template=model_card_template,
-            repo_url=repo_url,
-            docs_url=docs_url,
-            model_card_data=ModelCardData(
-                languages=languages,
-                library_name=library_name,
-                license=license,
-                license_name=license_name,
-                license_link=license_link,
-                pipeline_tag=pipeline_tag,
-                tags=tags,
-            ),
-        )
+
+        # Initialize MixinInfo if not existent
+        if getattr(cls, "_hub_mixin_info", None) is None:
```

Comment:
nit, would it be better to use `hasattr`?
```suggestion
        if not hasattr(cls, "_hub_mixin_info"):
```